boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280_cpurad.dts

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nordic/nrf9280_cpurad.dtsi>
+#include "nrf9280pdk_nrf9280-memory_map.dtsi"
+#include "nrf9280pdk_nrf9280-ipc_conf.dtsi"
+#include "nrf9280pdk_nrf9280-pinctrl.dtsi"
+
+/delete-node/ &cpuapp_cpuppr_ipc;
+/delete-node/ &cpuapp_cpusys_ipc;
+/delete-node/ &cpusec_cpuapp_ipc;
+
+/ {
+	compatible = "nordic,nrf9280pdk_nrf9280-cpurad";
+	model = "Nordic nRF9280 DK nRF9280 Radio MCU";
+
+	chosen {
+		zephyr,console = &uart135;
+		zephyr,code-partition = &cpurad_slot0_partition;
+		zephyr,flash = &mram1x;
+		zephyr,sram = &cpurad_ram0;
+		zephyr,shell-uart = &uart135;
+		zephyr,ieee802154 = &cpurad_ieee802154;
+		zephyr,bt-hci-ipc = &ipc0;
+		nordic,802154-spinel-ipc = &ipc0;
+	};
+	aliases {
+		ipc-to-cpusys = &cpurad_cpusys_ipc;
+		resetinfo = &cpurad_resetinfo;
+	};
+};
+
+&shared_ram3x_region {
+	status = "okay";
+};
+
+&cpuapp_cpurad_ram0x_region {
+	status = "okay";
+};
+
+&cpurad_bellboard {
+	status = "okay";
+	interrupts = <96 NRF_DEFAULT_IRQ_PRIORITY>;
+	interrupt-names = "irq0";
+	/* irq0: 0: cpurad-cpusec, 6: cpurad-cpusys, 12: cpurad-cpuapp */
```

Comment:
Like in cpuapp, I would change the comment to cpusec->cpurad, ... to indicate the signal direction.

Reply:
Thank you, I have updated this and also rephrased it to make it more explicit.